app/views/home/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<main class='splash-screen'>
+  <%= image_tag('spendwise-logo.png', alt: 'Spendwise Logo', class: 'logo') %>
+  <div class='buttons-wrapper mb-1'>
+    <% if user_signed_in? %> 
+      <%= button_to "Sign out", destroy_user_session_path, method: :delete %> 
+    <% else %>
+      <%= link_to "Log in", new_user_session_path, class: "btn btn-primary" %> 
+      <%= link_to "Sign up", new_user_registration_path, class: "btn btn-secondary" %>     
+    <% end %>
+  </div>
```

Comment:
- [ ] kindly as the design was given on mobile version consider making sure that your mobile design looks exactly as the on provided in the design here am talking to these buttons  please include the space around the button to meet the design requirement